core/templates/components/oppia-angular-root.component.ts

```diff
@@ -496,9 +496,6 @@ import { StateObjectFactory } from 'domain/state/StateObjectFactory';
 import { StateParamChangesService } from
   // eslint-disable-next-line max-len
   'components/state-editor/state-editor-properties-services/state-param-changes.service';
-import { StatePropertyService } from
-  // eslint-disable-next-line max-len
-  'components/state-editor/state-editor-properties-services/state-property.service';
```

Comment:
I removed it from here because it should be used directly but should be inherited first.